app/controllers/concerns/lead_provider_authenticatable.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module LeadProviderAuthenticatable
+  extend ActiveSupport::Concern
+  include ActionController::HttpAuthentication::Token::ControllerMethods
+
+  included do
+    before_action :authenticate
+  end
+
+  def authenticate
```

Comment:
Done

Reply:
I think it's probably because of the way that RSpec and Papertrail interact. Checked against the existing papertrail implementation and it's the same there. As we're not actually doing anything with it now, I'm happy to leave this as is for now. We probably won't need whodunnit when we get to the point of making sure that the lead_provider who created the record is the lead_provider who is triggering that state and making sure that the participant is counted for the correct lead_provider.